app/src/main/java/org/zalando/nakadi/filters/RequestRejectedFilter.java

```diff
@@ -0,0 +1,23 @@
+package org.zalando.nakadi.filters;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.security.web.firewall.RequestRejectedException;
+import org.springframework.stereotype.Component;
+import javax.servlet.http.HttpServletResponse;
+
+@Aspect
```

Comment:
i am not against aspects, but i am not really comfortable with it and it is not used in the project. maybe we could try to avoid using it.

Reply:
@adyach I have addressed your comment. I kept the same approach by intercepting doFilter method, but now the implementation is a Bean.

Reply:
@burimshala nice ! i like it :)